labs/blockchain-explorer-iroha.md

```diff
@@ -0,0 +1,17 @@
+# Lab Name
+[Integration of Hyperledger Iroha into Hyperledger Explorer tool](https://wiki.hyperledger.org/display/INTERN/Integration+of+Hyperledger+Iroha+into+Hyperledger+Explorer+tool)
+
+# Short Description
+Internship project: [Integration of Hyperledger Iroha into Hyperledger Explorer tool](https://wiki.hyperledger.org/display/INTERN/Integration+of+Hyperledger+Iroha+into+Hyperledger+Explorer+tool).  
+Fork of [hyperledger/blockchain-explorer](https://github.com/hyperledger/blockchain-explorer) with support for Hyperledger Iroha.
+
+# Scope of Lab
+Code of [hyperledger/blockchain-explorer](https://github.com/hyperledger/blockchain-explorer) supported only Hyperledger Fabric and needed refactoring.  
+This fork supports only Hyperledger Iroha. It rebuilds backend part from scratch and refactors frontend part.
+
+# Initial Committers
+- https://github.com/turuslan
+- https://github.com/Pobepto
+
```

Comment:
The `Pre-existing repository` section seems to be missing. Could you please add explanation on whether this repository will be used for the development or the code is available in a fork already?